app/frontend/pgbus/tailwind.css

```
@@ -4,18 +4,35 @@
   & tbody tr {
     @media (max-width: 1023px) {
       display: block;
+      width: 100%;
+      max-width: 100%;
+      box-sizing: border-box;
       margin-bottom: 0.75rem;
       border-radius: 0.5rem;
       padding: 0.75rem;
       border: 1px solid #e5e7eb;
+      overflow-wrap: anywhere;
+      word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)